### ISSUE TITLE
only space not working

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mblanchette/maven-java-formatter-plugin#31]
```
I want to format code by replace all tab to space
In Eclipse Kepler, Indentation -> tab policy  => use space only
After export xml file, I found two properties
<setting id="org.eclipse.jdt.core.formatter.tabulation.char" value="space"/>
<setting 
id="org.eclipse.jdt.core.formatter.use_tabs_only_for_leading_indentations" 
value="true"/>

I think they are conflict settings.
There two way to format my code 
===========Test1: use formatter in Eclipse===========
Expected output:
all tabs are replace to spaces
Actual output:
all tabs are replace to spaces

Steps to reproduce:
1. mouse right click
2. source
3. format


===========Test2: use maven plugin===========
Expected output:
all tabs are replace to spaces
Actual output:
all tabs are replace to spaces, !!!except leading_indentation ones!!!

Steps to reproduce:
1. run plugin

Plugin version: 0.4
Maven version: 3.1.0
Java version: 1.7_x64
Eclipse version: Kepler
OS: Win7_x64

Additional details:
I have tried to set false value to use_tabs_only_for_leading_indentations.
(in text editor, I can not find a way to do this in eclipse )
The result of the Test2 comes to right.

I guess eclipse will ignore use_tabs_only_for_leading_indentations, if 
tabulation.char equals space.


```

Original issue reported on code.google.com by `limhi....@gmail.com` on 30 Oct 2013 at 3:02

Attachments:
* [eclipse-formatter-config.xml](https://storage.googleapis.com/google-code-attachments/maven-java-formatter-plugin/issue-31/comment-0/eclipse-formatter-config.xml)

**Comment by hazendaz** [comment]
I believe this is fixed in 0.5 version which will be getting a PR shortly.

**Comment by mblanchette** [comment]
See merge in #37
